src/prefpicker/templates/browser-fuzzing.yml

```diff
@@ -1081,6 +1081,11 @@ pref:
     variants:
       default:
       - ''
+  webgl.colorspaces.prototype:
```

Comment:
Is there a `review_on_close` bug available?

Reply:
Resolved in https://github.com/MozillaSecurity/prefpicker/commit/6fb1d922de47cba1fc2e2052a02a9ebd1db5a5ff.